packages/react/src/lib/accessibility/Behaviors/Tree/treeItemBehavior.ts

```diff
@@ -20,7 +20,7 @@ const treeItemBehavior: Accessibility<TreeItemBehaviorProps> = props => ({
     root: {
       role: 'none',
       ...(props.hasSubtree && {
-        'aria-expanded': props.open,
+        'aria-expanded': props.open ? 'true' : 'false',
```

Comment:
There are a few instances in Stardust where we pass a `boolean` to `aria-expanded`. Let's report it as an issue?

Reply:
hmm let me take a look

Reply:
for consistency I made this one boolean as well. what do you think?

Reply:
Looks good to me